app/controllers/Application.java

```diff
@@ -514,26 +550,18 @@ public static Result flowHistory() {
       // if graph type is time or resources, we don't need the result from APP_HEURISTIC_RESULTS
       results = AppResult.find.select(
           AppResult.getSearchFields() + "," + AppResult.TABLE.FLOW_EXEC_ID + "," + AppResult.TABLE.FLOW_EXEC_URL + ","
-              + AppResult.TABLE.JOB_DEF_ID + "," + AppResult.TABLE.JOB_DEF_URL + "," + AppResult.TABLE.JOB_NAME)
-          .where()
-          .eq(AppResult.TABLE.FLOW_DEF_ID, flowDefPair.getId())
-          .order()
-          .desc(AppResult.TABLE.FINISH_TIME)
-          .setMaxRows(JOB_HISTORY_LIMIT)
-          .findList();
+              + AppResult.TABLE.JOB_DEF_ID + "," + AppResult.TABLE.JOB_DEF_URL + "," + AppResult.TABLE.JOB_NAME).where()
```

Comment:
Autoformatting by idea changed it. I'll change it back. Thanks for pointing out.